src/main/java/com/workmatch/resources/CandidatoResource.java

```diff
@@ -0,0 +1,17 @@
+package com.workmatch.resources;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/candidatos")
+public class CandidatoResource {
+
+	@RequestMapping(method = RequestMethod.GET)
+	public String listarCandidato() {
+
+		return "Testando REST...";
+	}
```

Comment:
Uma boa prática é você retornar o objeto `ResponseEntity<T>` em vez do tipo diretamente.  Só não lembro o pacote dele. Mas o Intellij vai importar ele para você automaticamente.

Fazendo assim, você consegue controlar qual status http você irá retornar no endpoint.

```suggestion
	public ResponseEntity<String> listarCandidato() {

		return EntityResponse.ok("Testando REST...");
	}
```

Você pode ler mais sobre ele [aqui](https://www.baeldung.com/spring-response-entity).

No seu exemplo aí, não vai fazer diferença porque você está retornando o código 200, mas como outras pessoas podem usar isso como exemplo, já é bom ir acostumando a usar esse cara.